sway/commands/focus.c

```diff
@@ -37,6 +37,8 @@ static struct sway_node *get_node_in_output_direction(
 		struct sway_output *output, enum wlr_direction dir) {
 	struct sway_seat *seat = config->handler_context.seat;
 	struct sway_workspace *ws = output_get_active_workspace(output);
+	if (!ws) {
```

Comment:
Missing if body?

Reply:
Not sure what happened there. Should be fixed